resources/language/resource.language.es_es/strings.po

```diff
@@ -264,7 +266,7 @@ msgstr "diariamente"
 
 msgctxt "#30068"
 msgid "every other day"
-msgstr "cada dos días"
+msgstr "en días alternos"
 
```

Comment:
DON'T AGREE. My previous translation is perfectly clear for that option.

Reply:
AGREE